### PR TITLE
Aml 3649 finance db untaggle

### PR DESCRIPTION
**Pull request by tom-gough**

### CODE REVIEW

File: src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetPayeForAccount_ByRef.sql [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	@Ref NVARCHAR(16)
 AS
 Select TOP 1
-	paye.Ref,
+	paye.EmpRef as Ref,
```
Comment (tom-gough) [inline src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetPayeForAccount_ByRef.sql]: @MichaelYoung1981 please confirm this is correct

File: src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CreateLegalEntityCommandTests/WhenTheUserIsNotWhitelistedForExpressionOfInterest.cs [hunk 2]
```diff
@@ -15,7 +14,7 @@ public override void Arrange()
         {
             base.Arrange();
 
-            AuthorizationService.Setup(x => x.IsAuthorized(FeatureType.ExpressionOfInterest)).Returns(false);
+            AuthorizationService.Setup(x => x.IsAuthorized("Feature.ExpressionOfInterest")).Returns(false);
```
Comment (tom-gough) [inline src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CreateLegalEntityCommandTests/WhenTheUserIsNotWhitelistedForExpressionOfInterest.cs]: should this be EmployerFeature. like the other one?

Comment (MichaelYoung1981) [reply src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CreateLegalEntityCommandTests/WhenTheUserIsNotWhitelistedForExpressionOfInterest.cs]: It should

Comment (CodeWolf86) [reply src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CreateLegalEntityCommandTests/WhenTheUserIsNotWhitelistedForExpressionOfInterest.cs]: Updated in next commit.

**Pull request merged**